dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/ProgramTrackedEntityAttribute.java

```diff
@@ -68,6 +68,8 @@ public class ProgramTrackedEntityAttribute extends BaseIdentifiableObject
 
   private Boolean searchable = false;
 
+  private boolean skipIndividualAnalytics;
```

Comment:
I am wondering, would it be better to just call this `skipAnalytics` ? 

I think I get the purpose of just skipping this very attribute in analytics export, vs. the entire program.

For `ProgramStageDataElement` we also have `skipAnalytics`, so this will make this consistent.

I general this is a very nice new feature/option which is great for avoiding exposure of sensitive data.

Reply:
@larshelge  This flag does not behave the same way as `skipAnalytics` in `ProgramStageDataElement` 
The intention of this new flag `skipIndividualAnalytics` is to allow aggregation to happen for these attributes, but blocking these attributes from being fetched individually through analytics/query api (from Line Listing App, Maps App) and so on. The attribute values will still be consumed for aggregate analytics.